src/pages/settings/ExitSurvey/ExitSurveyConfirmPage.tsx

```diff
@@ -0,0 +1,107 @@
+import useNetwork from '@hooks/useNetwork';
+import useThemeStyles from '@hooks/useThemeStyles';
+import Navigation from '@navigation/Navigation';
+import type {SettingsNavigatorParamList} from '@navigation/types';
+import variables from '@styles/variables';
+import * as ExitSurvey from '@userActions/ExitSurvey';
+import * as Link from '@userActions/Link';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import ROUTES from '@src/ROUTES';
+import type SCREENS from '@src/SCREENS';
+import type {ExitReason, ExitSurveyReasonForm} from '@src/types/form/ExitSurveyReasonForm';
+import EXIT_SURVEY_REASON_INPUT_IDS from '@src/types/form/ExitSurveyReasonForm';
+import ExitSurveyOffline from './ExitSurveyOffline';
+
+type ExitSurveyConfirmPageOnyxProps = {
+    exitReason?: ExitReason;
+    isLoading: OnyxEntry<boolean>;
+};
+
+type ExitSurveyConfirmPageProps = ExitSurveyConfirmPageOnyxProps & StackScreenProps<SettingsNavigatorParamList, typeof SCREENS.SETTINGS.EXIT_SURVEY.CONFIRM>;
+
+function ExitSurveyConfirmPage({exitReason, isLoading, route, navigation}: ExitSurveyConfirmPageProps) {
+    const {translate} = useLocalize();
+    const {isOffline} = useNetwork();
+    const styles = useThemeStyles();
+
+    const getBackToParam = useCallback(() => {
+        if (isOffline) {
+            return ROUTES.SETTINGS;
+        }
+        if (exitReason) {
+            return ROUTES.SETTINGS_EXIT_SURVEY_RESPONSE.getRoute(exitReason, ROUTES.SETTINGS_EXIT_SURVEY_REASON);
+        }
+        return ROUTES.SETTINGS;
+    }, [exitReason, isOffline]);
+    const {backTo} = route.params;
+    useEffect(() => {
+        const newBackTo = getBackToParam();
+        if (backTo === newBackTo) {
+            return;
+        }
+        navigation.setParams({
+            backTo: newBackTo,
+        });
+    }, [backTo, getBackToParam, navigation]);
+
+    return (
+        <ScreenWrapper testID={ExitSurveyConfirmPage.displayName}>
+            <HeaderWithBackButton
+                title={translate('exitSurvey.header')}
+                onBackButtonPress={() => Navigation.goBack(backTo)}
+            />
+            <View style={[styles.flex1, styles.justifyContentCenter, styles.alignItemsCenter, styles.mh5]}>
+                {isOffline && <ExitSurveyOffline />}
+                {!isOffline && (
+                    <>
+                        <Icon
+                            src={MushroomTopHat}
+                            width={variables.mushroomTopHatWidth}
+                            height={variables.mushroomTopHatHeight}
+                        />
+                        <Text style={[styles.headerAnonymousFooter, styles.mt5, styles.textAlignCenter]}>{translate('exitSurvey.thankYou')}</Text>
+                        <Text style={[styles.mt2, styles.textAlignCenter]}>{translate('exitSurvey.thankYouSubtitle')}</Text>
+                    </>
+                )}
+            </View>
+            <FixedFooter>
+                <Button
```

Comment:
Coming from #51145 (BZ Checklist)

We should add pressOnEnter to maintain consistency with the previous step, where pressing enter submits to the next step.